src/status_im2/common/toasts/view.cljs

```diff
@@ -0,0 +1,98 @@
+(ns status-im2.common.toasts.view
+  (:require
+   [quo2.components.notifications.toast :as quo2.toast]
+   [react-native.core       :as rn]
+   [react-native.gesture    :as gesture]
+   [react-native.reanimated :as reanimated]
+   [reagent.core            :as reagent]
+   [status-im.utils.utils   :as utils.utils]
```

Comment:
are we also aligning requires?

Reply:
not now, soon we'll be able to do this with zprint https://github.com/status-im/status-mobile/pull/14128